ex_feels/lib/ex_feels_web/templates/layout/app.html.eex

```diff
@@ -21,4 +24,594 @@
 
     <script src="<%= static_path(@conn, "/js/app.js") %>"></script>
   </body>
+  
+  <script src="https://d3js.org/d3.v3.min.js"></script>
+
+  <script>
+    /* */
+
+    var rawDataJSON =  [
```

Comment:
Lets move all the chart javascript into a file, `chart.js`. The process is the same as you did for the css.

Reply:
Just a thought toward the future here... We may eventually want more than one type of chart. Maybe a more specific name like `timefeels.js` is better? Autocorrelation and/or correlation between btc price and feels would be similar plot types but not necessarily the same colors etc. I don't know JavaScript so disregard if I'm just a brainlet. Also I think a wordcloud would be a sweet second viz... just throwing that out there

Reply:
A more specific name sounds good to me.  Lets make sure that the chart code file and the chart css file is named the same though.